features/guidance/notification.feature

```diff
@@ -0,0 +1,65 @@
+@routing  @guidance @mode-change
+Feature: Notification on turn onto mode change
+
+    Background:
+        Given the profile "car"
+        Given a grid size of 20 meters
+
+    Scenario: Turn onto a Ferry
+        Given the node map
+            |   |   |   |   |   | f |
+            | b | c |   |   | d | e |
+            |   |   |   |   |   |   |
+            | a |   |   |   |   | g |
+
+        And the ways
+            | nodes | highway | route | name  |
```

Comment:
apart from the ferry (where d is just a superfluous node), all roads are simply two-hop roads. Very short ones at that. If you don't want to have collapse interact here, we should at least use a grid distance 200 meters.

In addition, having names here for two coordinate roads only complicates reading the map, especially since the names don't have anything to do with the location. I find it very difficult to read these maps due to this, since it gives an additional information that is not actually useful.